app/import_functions.go

```diff
@@ -297,8 +297,9 @@ func (a *App) importUser(data *UserImportData, dryRun bool) *model.AppError {
 
 	var user *model.User
 	var err *model.AppError
```

Comment:
Is this `err` still used now?

Reply:
It is used much lower so I'll move the declaration below